app/models/item.rb

```diff
@@ -0,0 +1,19 @@
+class Item < ApplicationRecord
+  belongs_to :user
+  has_one_attached :images
+  extend ActiveHash::Associations::ActiveRecordExtensions
+  belongs_to :category
+  belongs_to :status
+  belongs_to :fee
+  belongs_to :prefecture
+  belongs_to :schedule
+  validates :category_id, :status_id, :fee_status_id, :prefecture_id, :schedule_delivery_id, numericality: { other_than: 1 }
+  with_options presence: true do
+    validates :name
+    validates :info
+    validates :price
+    validates :images
```

Comment:
```suggestion
    validates :image
```
上記のように、こちらのimagesも単数形へ修正しましょう。

Reply:
修正しました